itou/siaes/management/commands/import_siae.py

```diff
@@ -367,27 +77,26 @@ def fix_missing_external_ids(self):
                     siae.external_id = external_id
                     siae.save()
 
-    def siae_can_be_deleted(self, siae):
-        return siae.members.count() == 0 and siae.job_applications_received.count() == 0
-
     def delete_siae(self, siae):
-        assert self.siae_can_be_deleted(siae)
+        assert could_siae_be_deleted(siae)
         if not self.dry_run:
             siae.delete()
 
+    @timeit
     def delete_siaes_without_external_id(self):
         """
         Any siae which cannot be found in the latest ASP exports
         is a "ghost" siae which should be deleted.
```

Comment:
Comment peut-on se retrouver dans un état "ghost" ? Ça serait pas mal de l'expliquer ici.

Reply:
Oui, mais si une SIAE devient "ghost" *aprés* avoir eu une activité sur la plateforme ?

Reply:
```
    def delete_siaes_without_external_id(self):
        """
        Any siae which cannot be found in the latest ASP exports
        is a "ghost" siae which should be deleted.

        This can happen e.g. after every opening when old siaes imported back in 2019
        are no longer present in latest ASP exports.

        Of course we only delete it if it does not have any data,
        otherwise we just show a warning.
        """
```